runners/core-java/src/main/java/org/apache/beam/runners/core/SimpleDoFnRunner.java

```diff
@@ -942,7 +947,12 @@ public TimerMap timerFamily(String timerFamilyId) {
         TimerSpec spec =
             (TimerSpec) signature.timerFamilyDeclarations().get(timerFamilyId).field().get(fn);
         return new TimerInternalsTimerMap(
-            timerFamilyId, window(), getNamespace(), spec, stepContext.timerInternals());
+            timerFamilyId,
+            window(),
+            getNamespace(),
+            spec,
+            fireTimestamp(),
```

Comment:
ditto - this should be timestamp() not fireTimestamp()

Reply:
done